src/app/footer/footer.component.html

```diff
@@ -5,8 +5,8 @@
         <li>
           <span>
             <a href="https://github.com/brigadecore/kashti" target="_blank">KASHTI</a> and
-            <a href="https://github.com/brigadecore/brigade" target="_blank">BRIGADE</a> are licensed under the The
-            Apache 2.0 License.</span>
+            <a href="https://github.com/brigadecore/brigade" target="_blank">BRIGADE</a> are licensed under the The MIT
```

Comment:
This should actually stay `Apache-2.0`, right?

Reply:
Oh, you're right - seems like I missed cherry-picking that commit.
Nice catch, thanks!